packages/webviz-core/src/dataProviders/ZiplineItcDataProvider.js

```diff
@@ -202,6 +202,32 @@ export default class ZiplineItcDataProvider implements DataProvider {
       }
       return output;
     },
+    CHOSEN_AVOIDANCE_ACTION: async (baseTopic: string): Promise<{| timestamp: number, buffer: ArrayBuffer |}[]> => {
+      const { buffer, timestamps } = await this._getFile(baseTopic);
+      const parsedDefinition = parseMessageDefinition(this._typeInfoByType.CHOSEN_AVOIDANCE_ACTION.messageDefinition);
+      const reader = new MessageReader(parsedDefinition, { freeze: true });
+      const markers = [];
+      for (const timestamp of timestamps) {
+        const { start_n, start_e, start_d } = reader.readMessage(
+          Buffer.from(buffer, timestamp.offsetBegin, timestamp.offsetEnd - timestamp.offsetBegin)
+        );
+        var points = [];
+        var i;
+        for (i = 0; i < start_n.length; i++) {
```

Comment:
Nit, I think this is the idiomatic syntax
```suggestion
        for (var i = 0; i < start_n.length; i++) {
```

Reply:
Agreed. Changed thanks!